modules/text2vec-openai/clients/vectorizer.go

```diff
@@ -56,7 +56,8 @@ func buildUrl(config ent.VectorizationConfig) (string, error) {
 		queryParam := "api-version=2022-12-01"
 		return fmt.Sprintf("%s/%s?%s", host, path, queryParam), nil
 	}
-	host := "https://api.openai.com"
+
+	host := config.BaseURL
```

Comment:
I would change a logic here a little bit, to introduce this `baseURL` override in a non-breaking and safe way like so: 
```
host := "https://api.openai.com"
if config.BaseURL != "" {
  host = config.BaseURL
}
```